tests/installed/destructive/staged-deploy.yml

```diff
@@ -11,7 +12,7 @@
     orig_mtime=$(stat -c '%.Y' /sysroot/ostree/deploy)
     ostree admin deploy --stage staged-deploy
     new_mtime=$(stat -c '%.Y' /sysroot/ostree/deploy)
-    assert_not_streq "${orig_mtime}" "${new_mtime}"
```

Comment:
It'd be interesting to have a custom ansible module that wraps shell scripts in `set -xeuo pipefail` and imports the `libtest-core.sh` stuff.